### PR TITLE
Doc 4006 pc hotfix hci

### PR DESCRIPTION
**Pull request by netapp-pcarriga**
Updating links and removing SolidFire references

### CODE REVIEW

**Review by netapp-dbagwell — CHANGES_REQUESTED** [review]

File: docs/task_hcc_upgrade_element_software.adoc [hunk 1]
```diff
@@ -176,8 +174,9 @@ https://<ManagementNodeIP>/package-repository/1/
 |Your management node is within a dark site without external connectivity.
 a|
 . Download the storage upgrade package to a device that is accessible to the management node:
-* For NetApp HCI systems, go to the NetApp HCI software https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tab[download page] and download the latest storage node image.
-* For SolidFire storage systems, go to the Element software https://mysupport.netapp.com/site/products/all/details/element-software/downloads-tab[download page] and download the latest storage node image.
++
+For NetApp HCI systems, go to the NetApp HCI software https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tab[download page^] and download the latest storage node image.
```
[the remark this answers is not in the document]
Comment (netapp-pcarriga) [reply docs/task_hcc_upgrade_element_software.adoc:178]: @netapp-dbagwell Typo corrected

File: docs/rn_compute_firmware_2.27.adoc [hunk 2]
```diff
@@ -100,11 +100,7 @@ NOTE: Before upgrading to version 2.27, make sure you check the current ESXi dri
 While NetApp Deployment Engine 1.8P1 does not deploy ESXi version 7.0, ESXi 7.0 is supported as a self-install or manual upgrade option. When you manually install or upgrade host OSes, NetApp cannot control the drivers and firmware for these OSes.
 
 == Download Compute Firmware Bundle 2.27
-To download the Compute Firmware Bundle:
-
-. Go to https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tab[the NetApp HCI downloads page^].
-. Choose *Compute_Firmware_Bundle* from the drop down list.
-. Click *Go*.
+To download the Compute Firmware Bundle, go to the https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tab/download/62542/Compute_Firmware_Bundle[NetApp HCI Compute Firmware Bundle^] page.
```
[the remark this answers is not in the document]
Comment (netapp-pcarriga) [reply docs/rn_compute_firmware_2.27.adoc:103]: @netapp-dbagwell Response from Nicole "We don't have older versions available. Only the latest storage or compute firmware package is ever available.  Hence why we just update the one page and don't have a newer version on the product page."

File: docs/task_hcc_upgrade_element_software.adoc [hunk 3]
```diff
@@ -378,12 +377,10 @@ CAUTION: Aborting the cluster upgrade while a node is being updated might result
 
 .Steps
 
-. Download the storage upgrade package:
+. Download the storage upgrade package. For NetApp HCI systems, go to the NetApp HCI software https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tab[download page^] and download the latest storage node image to a device that is not the management node.
```
[the remark this answers is not in the document]
Comment (netapp-pcarriga) [reply docs/task_hcc_upgrade_element_software.adoc:380]: Done

File: docs/task_hcc_upgrade_management_node.adoc [hunk 4]
```diff
@@ -163,7 +163,7 @@ NOTE: Persistent volumes might require an additional network adapter if eth0 is
 .. Change the RAM of the management node VM from 12GB to 24GB RAM.
 .. Power on the management node VM.
 . Log in to the management node virtual machine using SSH or console access.
-. Download the management node ISO for https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tab[NetApp HCI] or https://mysupport.netapp.com/site/products/all/details/element-software/downloads-tab[Element] software from the NetApp Support Site to the management node virtual machine.
+. Download the management node ISO for https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tab[NetApp HCI] from the NetApp Support Site to the management node virtual machine.
```
[the remark this answers is not in the document]
Comment (netapp-pcarriga) [reply docs/task_hcc_upgrade_management_node.adoc:166]: Done

File: docs/task_mnode_install.adoc [hunk 5]
```diff
@@ -57,10 +57,7 @@ Prior to following this procedure, you should have an understanding of  link:con
 
 == Download ISO or OVA and deploy the VM
 
-. Download the OVA or ISO for your installation from the NetApp Support Site:
-+
-Element software: https://mysupport.netapp.com/site/products/all/details/element-software/downloads-tab
-NetApp HCI: https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tab
+. Download the OVA or ISO for your installation from the https://mysupport.netapp.com/site/products/all/details/netapp-hci/downloads-tabNetApp[NetApp HCI^] page on the NetApp Support Site.
```
[the remark this answers is not in the document]
Comment (netapp-pcarriga) [reply docs/task_mnode_install.adoc:60]: Done

File: docs/rn_compute_firmware_2.27.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (netapp-dbagwell) [reply docs/rn_compute_firmware_2.27.adoc:103]: Ok, that sounds familiar to me now. Thanks for confirming!

**Pull request merged**